### PR TITLE
feat(eslint-plugin-template): add cyclomatic-complexity rule

### PR DESCRIPTION
**Pull request by paulotokimatu**

### CODE REVIEW

File: packages/integration-tests/fixtures/angular-cli-workspace/.eslintrc.js [hunk 1]
```diff
@@ -193,6 +193,9 @@ module.exports = {
 
         // ORIGINAL tslint.json -> "template-no-negated-async": true,
         '@angular-eslint/template/no-negated-async': 'error',
+
+        // ORIGINAL tslint.json -> "template-cyclomatic-complexity": false,
```
Comment (JamesHenry) [inline packages/integration-tests/fixtures/angular-cli-workspace/.eslintrc.js]: Was this the case? I thought I had captured all the rules that were included in the original tslint.json...

**Comment by JamesHenry** [comment]
Great stuff thanks, the template piece was very complex to figure out and I'm delighted it's in a place now where others can contribute!

File: packages/integration-tests/fixtures/angular-cli-workspace/.eslintrc.js (hunk 1 above)
Comment (paulotokimatu) [reply packages/integration-tests/fixtures/angular-cli-workspace/.eslintrc.js]: You are right, there is just no mention of the rule. I will remove it.

**Comment by paulotokimatu** [comment]
> 
> 
> Great stuff thanks, the template piece was very complex to figure out and I'm delighted it's in a place now where others can contribute!

Yes, when you get a bit familiar it gets straightforward to work with, great work with the template parsing!

**Pull request merged**